tap_hubspot/hubspot.py

```diff
@@ -389,21 +427,6 @@ def get_companies(self):
             offset_key=offset_key,
         )
 
-    def get_contacts(self, start_date: datetime, end_date: datetime):
-        self.event_state["contacts_start_date"] = start_date
-        self.event_state["contacts_end_date"] = end_date
-        path = "/crm/v3/objects/contacts"
-        data_field = "results"
-        offset_key = "after"
-        replication_path = ["updatedAt"]
-        params = {"limit": 100, "properties": MANDATORY_PROPERTIES["contacts"]}
-        yield from self.get_records(
-            path,
-            replication_path,
-            params=params,
-            data_field=data_field,
-            offset_key=offset_key,
-        )
```

Comment:
I would keep this as the very last commit - just so we have a reference for what the old thing did for the rest of the commits